stdlib/slack/slack_test.go

```diff
@@ -232,6 +242,9 @@ data = "
 			if req.PostData.Attachments[0].Color != tc.color {
 				t.Errorf("got color %s, expected %s", req.PostData.Attachments[0].Color, tc.color)
 			}
+			if _, hasKey := req.RawData["as_user"]; hasKey {
```

Comment:
I would suggest a comment here that this part of the test can be removed one year from today.

Reply:
I can add that comment, though I should note that this isn't quite the same as just e.g. asserting some random nonsense parameter wasn't included; you can include arbitrary nonsense in your requests and Slack will just ignore it. `as_user` is a specific documented parameter of their API that, if included in your requests, will cause the request to fail (assuming you aren't using certain legacy auth flows).